src/components/List.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ListItem from './ListItem';
+import classes from './List.module.css';
+import { useGlobalContext } from '../context';
+
+const List = () => {
+  const { list } = useGlobalContext();
+  {
+    if (list.length === 0)
+      return (
+        <div>
+          <h3>Empty List</h3>
+        </div>
+      );
+    return (
+      <ul className={classes.list}>
+        {list.map((el) => {
+          return <ListItem item={el} key={el.id} />;
+        })}
+      </ul>
+    );
+  }
```

Comment:
I think it would be good to unnest this block of code for readability :nesting_dolls:
> Nested block is redundant `no-lone-blocks`